source/OrderedTypes/PredicativeLFP.lagda

```diff
@@ -0,0 +1,1637 @@
+Ian Ray 01/09/2023.
+
+We formalize Curi's notion of abstract inductive definition (in CZF) within
+the context of a sup-lattice L with small basis B (and β : B → L). An abstract
+inductive defintion is a subset ϕ : B × L → Prop which can be thought of as a
```

Comment:
“definition”

Reply:
I would say `Ω` instead of `Prop` since we don’t use the latter term in TypeTopology.

Reply:
I understand the criticism but would like notes to be readable without much knowledge of the library (since notes aren't linked to other files). Does this make sense? If you still feel it better to change to omega then I will consider it. @tomdjong thoughts?